app/src/main/java/com/sun/englishlearning/screen/me/MeFragment.kt

```diff
@@ -46,8 +49,8 @@ class MeFragment : BaseFragment<FragmentMeBinding>() {
         if (user!= null) {
             // User is logged in: display personal information
             viewBinding.layoutUserProfile.visibility = View.VISIBLE
-            viewBinding.tvUserName.text = user.displayName
-            viewBinding.tvUserEmail.text = user.email
+//            viewBinding.tvUserName.text = user.displayName
+//            viewBinding.tvUserEmail.text = user.email
```

Comment:
```suggestion
//            viewBinding.tvUserName.text = user.displayName
//            viewBinding.tvUserEmail.text = user.email
```
sao lại comment đoạn code này lại?

Reply:
à e mới làm khung giao diện e chưa add dữ liệu vào view ạ